examples/multi_window_ref_app/lib/app/window_controller_render.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'regular_window_content.dart';
+import 'window_manager_model.dart';
+import 'window_settings.dart';
+
+class WindowControllerRender extends StatelessWidget {
+  WindowControllerRender(
+      {required this.controller,
+      required this.onDestroyed,
+      required this.onError,
+      required this.windowSettings,
+      required this.windowManagerModel,
+      required this.key});
+
+  final WindowController controller;
+  final VoidCallback onDestroyed;
+  final VoidCallback onError;
+  final WindowSettings windowSettings;
+  final WindowManagerModel windowManagerModel;
+  final Key key;
```

Comment:
Since `key` is already provided by `StatelessWidget`, could we just use `super.key`?